app/views/_shared/_privacy-notice.html

```diff
@@ -1,39 +1,65 @@
-{% filter markdownToHTML %}
```

Comment:
Do the changes here just bring the privacy notice inline with live, or is it more than that?